miden-tx/src/tests/kernel_tests/test_faucet.rs

```diff
@@ -333,19 +345,20 @@ fn test_burn_fungible_asset_fails_not_faucet_account() {
     let code = format!(
         "
         use.kernel::prologue
-        use.miden::faucet
+        use.test::account
 
         begin
             exec.prologue::prepare_transaction
             push.{FUNGIBLE_ASSET_AMOUNT}.0.0.{ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN_1}
-            exec.faucet::burn
+            call.account::burn
         end
         "
     );
 
     let process = tx_context.execute_code(&code);
 
-    assert!(process.is_err());
+    // ERR_FUNGIBLE_ASSET_MISMATCH
+    assert_execution_errored(process, 0x00020039);
```

Comment:
I'm not entirely sure all these error codes are correct and in line of what was being returned originally. I mean to double check this soon